src/lib/realtime.js

```diff
@@ -9,7 +9,7 @@ const APPS_DOCTYPE = 'io.cozy.apps'
  * @param {object}
  * @returns {Promise}
  */
-async function initializeRealtime({ getApp, onCreate, onDelete, cozyClient }) {
+function initializeRealtime({ getApp, onCreate, onDelete, cozyClient }) {
```

Comment:
If so, you probably should remove the `await` in client-internal.js`

Reply:
Bien vu, merci